main/client/main.css

```diff
@@ -89,3 +122,12 @@ body {
     position: absolute;
     width: 1px;
 }
+
+/*
+ * Clear all helper. 
+ */
+
+.clearfix {
```

Comment:
Bootstrap has a .[clearfix class](https://github.com/twbs/bootstrap/blob/v2.3.0/less/mixins.less). Please be certain that we want to override that class, or remove it from this PR.